From 27bac9cc729aae02702e2ce9ce047d3c9874a666 Mon Sep 17 00:00:00 2001
From: Jernej Jakob <jernej.jakob@gmail.com>
Date: Wed, 28 Jun 2023 15:41:43 +0200
Subject: [PATCH scute] doc: yat2m-stamp should depend on version.texi

* doc/Makefile.am (yat2m-stamp): depend on version.texi
--
Fixes "yat2m: can't open include file './version.texi': No such
file or directory"

https://lists.gnupg.org/pipermail/gnupg-devel/2018-April/033612.html

See-commit: 8ef76d9373db16f1ca176f9c87c8dac2758c8b85
Signed-off-by: Jernej Jakob <jernej.jakob@gmail.com>
---
 doc/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/Makefile.am b/doc/Makefile.am
index 42dfdd0..ed7c194 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -50,7 +50,7 @@ myman_pages   = scute.7
 man_MANS = $(myman_pages)
 
 
-yat2m-stamp: $(myman_sources)
+yat2m-stamp: $(myman_sources) $(srcdir)/version.texi
 	@rm -f yat2m-stamp.tmp
 	@touch yat2m-stamp.tmp
 	for file in $(myman_sources) ; do \
-- 
2.39.3

